docs/Hooks.md

```diff
@@ -121,6 +132,25 @@ fastify.addHook('preHandler', (request, reply, next) => {
 
 *The error will be handled by [`Reply`](https://github.com/fastify/fastify/blob/master/docs/Reply.md#errors).*
 
+#### The `onError` Hook
+
+This hook is useful if you need to do some custom error logging or add some specific header in case of error.<br/>
+It is not intended for changing the error, and calling `reply.send` will throw an exception.<br/>
+This hook will be executed only after the `customErrorHandler` has been executed.<br/>
+**Notice:** if you pass an error to `next` it will not be handled.
+
+```js
+fastify.addHook('onError', (request, reply, error, next) => {
+  apm.sendError(error)
```

Comment:
can you add a comment saying that apm stands for Application Performance Monitoring?